tonic-reflection/src/server/mod.rs

```diff
@@ -1,11 +1,307 @@
-pub use crate::pb::v1::server_reflection_server::{ServerReflection, ServerReflectionServer};
-
-use prost::DecodeError;
+use std::collections::HashMap;
 use std::fmt::{Display, Formatter};
+use std::sync::Arc;
+
+use prost::{DecodeError, Message};
+use prost_types::{
+    DescriptorProto, EnumDescriptorProto, FieldDescriptorProto, FileDescriptorProto,
+    FileDescriptorSet,
+};
+use tonic::Status;
 
 mod v1;
+pub use v1::{ServerReflection, ServerReflectionServer}; // For backwards compatibility
```

Comment:
How about adding a document to guide to the new place of these reexport.

Reply:
I guess we currently don't have such a new place. I guess we can make `v1 and `v1alpha` `pub` in `server`?